Header header
string unique_name
geometry_msgs/Pose2D pose
geometry_msgs/Twist velocity
geometry_msgs/PoseStamped[] route
